packages/concerto-core/test/data/decoratorcommands/test.cto

```diff
@@ -1,7 +1,11 @@
 namespace test@1.0.0
+import concerto@1.0.0.Decorator
```

Comment:
Interesting, should this be implicitly in-scope? For comparison, what's the behaviour for `Concept`?
e.g. `concept X extends Concept`